README.md

```diff
@@ -32,6 +34,77 @@ TODO: finish readme later
 * **DO NOT PUSH TO MASTER BRANCH!** Push to a new branch (e.g username-whatyouchanged and create a PR)
 
+## Setup instructions
+So, Karen Bot is quite tightly integrated with my *private* API, which is good & bad news. Good news is that a lot of strain is taken off of Karen and put on my API, while the bad news is that someone without API access can't really run Karen that good. Fortunately, I have designed a lot of the API to have as little server side processing as possible, ensuring a ~~tedious~~ simple transition to your own API or just saving files on your computer!
+So this section will walk you through the necessary things to get Karen Bot atleast slightly functional :)
+
+### Discord token
+This is going to be super simple
+First, go to [discord.com/developers/applications](https://discord.com/developers/applications) and click on the button "New application"
+![How to create new application](https://cdn.exerra.xyz/files/png/tutorials/discord_bot_token/create_application.png)
+
+Then, name your application
+![How to name application](https://cdn.exerra.xyz/files/png/tutorials/discord_bot_token/name_application.png)
+
+Great! Now you have an application, but not a bot! To get a bot, create on the "Bot" tab on the sidebar, then when it asks you if you *really* want to create a bot, just say yes!
+![How to create bot](https://cdn.exerra.xyz/files/png/tutorials/discord_bot_token/click_on_bot.png)
+
+Wow, you can follow simple instructions, great! To get the token, click on "Copy" under the token while on the Bot tab. Now, be careful where you put it since tokens are basically like usernames and passwords for bots.
+![How to copy token](https://cdn.exerra.xyz/files/png/tutorials/discord_bot_token/copy_token.png)
+
+Aaand you're done with the token part, congratz! 🎉
+
+### Spotify OAuth token
+First, go to [developer.spotify.com/dashboard/applications](https://developer.spotify.com/dashboard/applications) and log in/sign up (if you don't have a spotify account for some god awful reason)
+
+Then, click on the big green "Create application" button and give it a name & description
+![How to create app](https://cdn.exerra.xyz/files/png/tutorials/spotify_oauth/create_app.png)
+
+Now you have a shiny Spotify application, great! So next, click on "Show client secret". Make sure not to leak the secret since it is, well, a secret!
+![How to get OAuth token](https://cdn.exerra.xyz/files/png/tutorials/spotify_oauth/get_token.png)
+
+Oh hey, already at the end of this section! Good job, go pat yourself on the back 🥳
+
+### Genius OAuth token
+First, go to [genius.com/developers/](https://genius.com/developers/). Then create on the biggest button you can find, aka "Create an API client"
+![How to navigate the startpage](https://cdn.exerra.xyz/files/png/tutorials/genius_oauth/startpage.png)
+
+Then log in/sign up and you will be presented the API client creation screen. Fill in the name and if you want, the rest of the fields, then click "Save"

```

Comment:
I'd put here "you will be presented **with..**"

Reply:
Hmm, I think my version is a bit better
Idk how to describe but just imo my way feels "better"

Reply:
Smh, haha okie